### PR TITLE
Solve c

### PR DESCRIPTION
**Pull request by PaLeroy**
The problem was in the get_obs_size method

### CODE REVIEW

File: pymarl/config/envs/struct_c_10.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  n_comp: 10
+  discount_reward: 0.95
+  k_comp: 9
+  state_obs: True
+  state_d_rate: True
+  state_alphas: True
+  obs_d_rate: False
+  obs_multiple: False
+  obs_all_d_rate: False
+  obs_alphas: False
```
Comment (moratodpg) [inline pymarl/config/envs/struct_c_10.yaml]: Shall we add here "True"?

Comment (PaLeroy) [reply pymarl/config/envs/struct_c_10.yaml]: [depend on this discussion :D ](https://github.com/moratodpg/marl_struct/pull/16#discussion_r999435084)

**Comment by PaLeroy** [comment]
I think we can merge

File: pymarl/config/envs/struct_c_10.yaml (hunk 1 above)
Comment (PaLeroy) [reply pymarl/config/envs/struct_c_10.yaml]: 91bb5bc

**Review by moratodpg — APPROVED** [review]

**Pull request merged**